src/recommendations/Dockerfile

```diff
@@ -1,12 +1,11 @@
 FROM python:3.8-slim
 
-COPY /src/recommendations-service /app
-
 WORKDIR /app
 
-# Needed for Optimizely SDK (git ref in req.txt)
-RUN apt-get update && apt-get install -y g++ git
```

Comment:
I was partially wrong on this one. Removing this line entirely doesn't work for me. The image build fails. We still need g++ for an Optimizely SDK dependency but don't need git. So this line works for me.

`RUN apt-get update && apt-get install -y g++`